pkg/csi/service/vanilla/controller.go

```diff
@@ -531,23 +517,21 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 			}
 			queryResult, err := c.manager.VolumeManager.QueryVolume(ctx, queryFilter)
 			if err != nil {
-				log.Errorf("QueryVolume failed for volumeID: %s", volumeInfo.VolumeID.Id)
-				return nil, status.Error(codes.Internal, err.Error())
+				return nil, logger.LogNewErrorCodef(log, codes.Internal,
+					"QueryVolume failed for volumeID: %s, err: %+v", volumeInfo.VolumeID.Id, err.Error())
```

Comment:
I chose to use "err" with "%+v". It is not a log spew, since it is error case. And, we prefer to have more debug information for error case.

I prefer to fix one anti-pattern at a time, given the change is huge. Do you mind if we fix this err.Error() pattern in another effort? This will make code review easier.